### PR TITLE
Update e2e test

### PR DESCRIPTION
**Pull request by ricardobranco777**
This PR:

- Removes unused `requirements.txt` files that triggers automated PR's like https://github.com/ClusterLabs/hawk/pull/231
- Updates geckodriver
- Includes minor fixes

### CODE REVIEW

File: e2e_test/hawk_test_driver.py [hunk 1]
```diff
@@ -329,7 +329,7 @@ def test_remove_cluster(self, cluster_name):
                 print("ERROR: Couldn't find cluster [%s]. Cannot remove" % cluster_name)
                 return False
             elem.click()
-        time.sleep(2 * self.timeout_scale)
+        time.sleep(6)
```
Comment (MalloZup) [inline e2e_test/hawk_test_driver.py:332]: this `6` is a  magic number.
As an improvement you should consider to introduce a constant like

`BIG_TIMEOUT` whatever fits.

Se also line 347. Why do you increase here the timeout and not on 347 for example?

Comment (ricardobranco777) [reply e2e_test/hawk_test_driver.py:332]: > Se also line 347. Why do you increase here the timeout and not on line 347 for example?

Removing cluster takes more time.

Comment (ricardobranco777) [reply e2e_test/hawk_test_driver.py]: Fixed in last commit

**Comment by MalloZup** [comment]
thx @ricardobranco777  let's wait travis green then merging

**Pull request merged**